pkg/kubelet/lifecycle/handlers.go

```diff
@@ -30,24 +31,31 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 	"k8s.io/kubernetes/pkg/kubelet/util/format"
+	"k8s.io/kubernetes/pkg/probe/tcp"
 	"k8s.io/kubernetes/pkg/security/apparmor"
 )
 
+const (
+	TCPSocketTimeOut = 1 * time.Second
```

Comment:
How did you come up with 1 second? I'm not saying that 1 second or 10 seconds is wrong, I'd just like to know that this isn't an arbitrary made up number.

Reply:
Actually the default timeout to setup a tcp connection is 127 seconds. I think that's too long for a pod probe.